### DISCUSSION TITLE
no hint for function argument

### DISCUSSION THREAD

**Discussion by ssh352** [R-nvim/R.nvim#135] [Q&A] [answered]
after I migrated to R.nvim, no coc
<img width="359" alt="Screenshot 2024-05-11 at 11 43 41 PM" src="https://github.com/R-nvim/R.nvim/assets/41779116/45303a03-ab18-496f-aea2-5a78f139edca">

nvim-R + coc; I feel it is quite helpful to have hint for function argument. I am not sure if this functionality is provided by coc or by Nvim-R though. Is it possible to still have this feature after migrating to R.nvim? Thanks!

<img width="950" alt="Screenshot 2024-05-11 at 11 43 04 PM" src="https://github.com/R-nvim/R.nvim/assets/41779116/cd838be3-f8cf-486a-bd76-2da5bf0ae1d3">

**Comment by jalvesaq** [comment]
With [cmp-r](https://github.com/R-nvim/cmp-r), I get this:

![image](https://github.com/R-nvim/R.nvim/assets/891655/e92aec2b-67e3-47f5-872e-feef521718d3)

Perhaps, @PMassicotte knows how to get a result closer to what you previously had with `coc`.

**Reply by ssh352** [reply]
It actually looks pretty nice. I am curious why it needs a space to trigger the hint?

**Comment by jalvesaq** [comment]
> I am curious why it needs a space to trigger the hint?

It's because style guides suggest a space after the comma, but I can make the trigger characters customizable to allow removing the space and adding the comma.

**Reply by ssh352** [reply]
That would be even nicer.  I use styler to format code on write. styler adds missing spaces automatically.

**Comment by jalvesaq** [comment, marked as answer]
Option added: https://github.com/R-nvim/cmp-r/commit/efa34e762dea378cae27a9e47bdd95afb0bc8dfc

**Reply by ssh352** [reply]
tested, it works!

**Comment by PMassicotte** [comment]
Sorry, was out of town for few days. Glad you got it sorted out.
